phoenix-core/src/it/java/org/apache/phoenix/end2end/QueryLoggerIT.java

```diff
@@ -250,7 +250,44 @@ public void testWithLoggingOFF() throws Exception{
         assertFalse(foundQueryLog);
         conn.close();
     }
-    
+
+    @Test
```

Comment:
Please add a test for `Alter Table` too.

Reply:
Added